### DISCUSSION TITLE
[Dúvida] MacBook Air para compilação

### DISCUSSION THREAD

**Discussion by sscraul** [frontendbr/forum#1590] [Perguntas e Respostas]
Olá amigos! uma dúvida...um MacBook Air 2019 com as seguintes specs:
1.6GHz dual-core 8th-generation Intel Core i5 processor with Turbo Boost up to 3.6GHz
8GB 2133MHz LPDDR3 memory
256GB SSD

Dá pra trabalhar tranquilo com JS, Swift, React, HTML5?? 
Ou vou ter muita dor de cabeça? 
Um Macbook Pro está 500 USD mais caro e a grana tá curta 👎

**Comment by glauberm** [comment]
Dá e sobra.

**Comment by felipefialho** [comment]
Com certeza, rola fácil.

**Comment by cassiodourado1** [comment]
é tranquilo, eu usava o 2018 só que era o modelo de 128gb, acabei trocando ele  só pelo armazenamento que estava ficando quase sempre cheio.

**Comment by Trincazul** [comment]
eu uso um MacBook Air 2017 consigo usar tranquilamente pra tudo

**Comment by caioincau** [comment]
Vai de boa
